docs/apm/mobile-service.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[[mobile-service-overview]]
+=== Mobile service overview
+
+Selecting a mobile service brings you to the *Mobile service overview*.
+The *Mobile service overview* contains a wide variety of charts and tables that provide
+high-level visibility into how a mobile service is performing for your users--enabling you
+to make data-driven decisions about how to improve your user experience.
+
+For example, see:
+
+* Crash Rate (Crashes per minute) -- coming soon
+* Slowest App load time -- coming soon
+* Number of sessions
+* Number of HTTP requests
+* Map showing the total number of HTTP requests based on country and region
+* Most used devices, network connection type, OS version, and app version
+* Latency, throughput, and errors over time
+* Service dependencies
+
+All of this and more can help you understand how the performance of your backend impacts
+user experience on the frontend.
+
+[discrete]
+[[mobile-service-stats]]
+=== Quick stats
+
+Reduce your mean time to resolution time with insights into your slowest app load time and crash rate (coming soon).
+Visualize session and HTTP trends, and see where your users are located--enabling you to optimize your infrastructure.
```

Comment:
Does this sound accurate?

Reply:
@bmorelli25 - Alternate text proposal, "Understand the impact of slow app load times and variations in app. crash rate on user traffic." 
"Visualize session and HTTP trends, and see where your users are located--enabling you to optimize your  infrastructure deployment and routing topology."